python/py-pyqt5/Portfile

```diff
@@ -48,6 +48,8 @@ if {${subport} eq "${name}-common"} {
 } elseif {[string first "webengine" ${subport}] != -1} {
     PortGroup           qmake5 1.0
 
+    version             5.12.1
+    revision            0
```

Comment:
There is not yet a PyQtWebEngine 5.12.2 release; my guess is that their releases are no longer synchronized, so I have added a separate version + revision here for `py-pyqt5-webengine. Does this subport need to have its revision incremented when `py-pyqt5` is updated?

Reply:
If it links against the rest of Qt/PyQt, which seems likely, then that would be wise. It wouldn't be necessary if the API & ABI haven't of the dependencies haven't changed, but why take the risk?

Reply:
Well, at a minimum `port rev-upgrade` didn't complain for me. I dug a little more and ran `otool -L` on the .so files in `pyXX-pyqt5-webengine`; it appears to only link to libraries in `qt5` and not anything in `pyXX-pyqt5`, and all have `compatibility version` 5.12.0. Maybe someone else should confirm, but I'm now inclined to think it isn't necessary to increment the revision for the same major.minor version.

Reply:
Thanks for checking. No rev-bump necessary then.